includes/admin/upgrades/v3/class-data-migrator.php

```diff
@@ -556,15 +565,17 @@ public static function orders( $data = null ) {
 			'state'   => '',
 		) );
 
+		$address_created    = ! empty( $date_completed ) ? $date_completed : $date_created_gmt;
```

Comment:
Perhaps we should just use `$date_created_gmt` for this? The difference isn't huge (in most cases probably a matter of seconds/minutes), but technically the address was created as soon as the order was created -- not when the order was _completed_.